js/src/components/google-combo-account-card/connect-ads/connect-ads-footer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+
+/**
+ * Footer component.
+ *
+ * @return {JSX.Element} Footer component.
+ */
+const ConnectAdsFooter = ( { isConnected } ) => {
+	const text = isConnected
+		? __(
+				'Or, connect to a different Google Ads account',
+				'google-listings-and-ads'
+		  )
+		: __(
+				'Or, create a new Google Ads account',
+				'google-listings-and-ads'
+		  );
+	return <AppButton isTertiary>{ text }</AppButton>;
```

Comment:
@ankitrox Let's render `js/src/components/google-ads-account-card/disconnect-account.js` if the account is connected. If you can integrate the logic from `js/src/components/google-ads-account-card/connect-ads/index.js` for the `Or, create a new Google Ads account` button, that would be great!

Reply:
Let's handle the `Or, create a new Google Ads account` separately in #2603